contrib/proguard/src/Proguard.scala

```diff
@@ -25,10 +25,9 @@ trait Proguard extends ScalaModule {
 
   /**
    * The version of proguard to download from Maven.
-   *
-   * Note that currently this is the only available version.
+   * https://mvnrepository.com/artifact/com.guardsquare/proguard-base
    */
-  def proguardVersion: T[String] = T { "7.0.0" }
+  def proguardVersion: T[String] = T { "7.2.2" }
```

Comment:
That sounds sensible. I don't see breaking changes / compatiblity issues in their release notes, but this sounds good.

I see the twirl module has `def twirlVersion: T[String]`, i.e. the user has to take control.
I've been trying to specify the proguard version in tests too, similar to how [the twirl test does](https://github.com/com-lihaoyi/mill/blob/main/contrib/twirllib/test/src/HelloWorldTests.scala#L18). Even if it doesn't impact the test, it might be nice to check things pass when we specify latest version manually for tests.